api/v1alpha08/kogitoserverlessplatform_types.go

```diff
@@ -86,6 +86,8 @@ type KogitoServerlessPlatformSpec struct {
 	BuildPlatform api.PlatformBuildSpec `json:"platform,omitempty"`
 	// Configuration list of configuration properties to be attached to all the Workflow built from this Platform
 	Configuration ConfigurationSpec `json:"configuration,omitempty"`
+	// Operator version, by default aligned with the Kogito community version
+	OperatorVersion string `json:"operatorVersion,omitempty"`
```

Comment:
@radtriste see above the Ricardo statement

Reply:
@desmax74 wdym? This operator version should be something **internal** only. The user should be able to:

1) In devmode replace with their own image
2) If no image in the platform (or no platform at all), use the default. 
2.1) The default if the version has `snapshot` should be the nightly one, otherwise we use the release based on the `version/version.go`

We clarified this in the past meeting. Please update the JIRA and ask if you still have questions.

Reply:
Just to clarify: `OperatorVersion` SHOULD NOT be exposed to users whatsoever!

Reply:
@ricardozanini I was pointing your agreement with Davide about providing the possibility to change image and version like this https://github.com/kiegroup/kogito-serverless-operator/pull/80/files#diff-cc635941ea89d2ed576d6a3dc9e4c926858209209ae85f93d3502120369c432aR130-R146

Reply:
Yes, we can change the image version by just changing the image tag.